webservice/src/main/resources/app/scripts/containers/Streams/TestRunComponents/TestSourceNodeModal.jsx

```diff
@@ -179,7 +179,7 @@ class TestSourceNodeModal extends Component{
     const {showInputError,sourceNodeArr, testName} = this.state;
     let validate = false,validationArr = [];
     _.map(sourceNodeArr, (source,i) => {
-      if(source.records === '' || source.repeatTime === '' || source.repeatTime === undefined || source.records === undefined || testName === ''){
+      if(source.records === '' || source.repeatTime === '' || (source.repeatTime === undefined || source.repeatTime === 0 )|| source.records === undefined || testName === ''){
```

Comment:
It's ok to have 0 as Repeat when the user just wants to run the test on the provided record.
Let's not restrict the user to enter more than 0 as value.